### PR TITLE
Add script for releasing the lambda layer.

### PR DESCRIPTION
**Pull request by mirelap-amazon**
*Issue #, if available:*

*Description of changes:*

*Testing:*

Run ``python release_layer.py`` without actually publishing and checked the logs.

```
python release_layer.py 
...
creating build/lib/codeguru_profiler_agent
copying codeguru_profiler_agent/sampling_utils.py -> build/lib/codeguru_profiler_agent
...
Preparing to publish archive /var/folders/x_/_hj_vygs2757pbrtq8_7v_kxn9_25b/T/tmpisdjcdpb/layer.zip
Publish the layer? Check the archive before responding.  (y|n): n
Nothing was published.
```

```
ls -l /var/folders/x_/_hj_vygs2757pbrtq8_7v_kxn9_25b/T/tmpisdjcdpb
total 312
drwxr-xr-x  4 mirelap  ANT\Domain Users     128 12 Jan 17:09 layer
-rw-r--r--  1 mirelap  ANT\Domain Users  157953 12 Jan 17:09 layer.zip
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release_layer.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
+import codecs
+import re
+import subprocess
+import tempfile
+import shutil
+import os
+import json
+
+# The following values are used in the documentation, so any change of them requires updates to the documentation.
+LAYER_NAME = 'AWSCodeGuruProfilerPythonAgentLambdaLayerTestDev'
```
Comment (PapaPedro) [inline release_layer.py:10]: why does this have "TestDev" suffix, why not the official name?

File: release_layer.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            return False
+        else:
+            print('Please enter ' + answer_true + ' or ' + answer_false)
+
+
+def read(*parts):
+    return codecs.open(os.path.join(here, *parts), 'r').read()
+
+
+def find_version(*file_paths):
```
Comment (PapaPedro) [inline release_layer.py:58]: I think this deserves a comment, something like
"""
To keep track of which PyPi version corresponds to which layer version we cross reference the PyPi version in the layer description. This function is used to read the PyPi directly from the source code.
"""

File: release_layer.py (hunk 1 above)
Comment (mirelap-amazon) [reply release_layer.py:10]: Totally by mistake, I used it for testing and forgot it there, I will update it, thanks!

File: release_layer.py (hunk 2 above)
Comment (mirelap-amazon) [reply release_layer.py:58]: Yes, I will add it.

**Review by PapaPedro — APPROVED** [review]

**Pull request merged**